Packs/CommonScripts/Scripts/FeedRelatedIndicators/FeedRelatedIndicators.py

```diff
@@ -0,0 +1,48 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def feed_related_indicator():
```

Comment:
pass `indicator` as argument to this func so you won't have to mock it in the test